hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -610,7 +610,7 @@ tests:
   input_series:
   - series: 'kubevirt_hco_system_health_status'
     # time:  0     1     2     3     4     5     6     7     8     9     10     11
-    values: "0     0     0     0     1     1     1     1     2     2      2      2"
+    values: "1     1     1     1     2     2     2     2     3     3      3      3"
```

Comment:
@machadovilaca There should no be a value 3.
Values of this metric should be healthy (0), warning (1), or error (2).